configGenerator.js

```diff
@@ -51,12 +51,14 @@ try {
   // Write the substituted data to config.json
   try {
     fs.writeFileSync(currentConfigFilePath, substitutedData, 'utf-8');
-    console.log('Configuration file written in current directory.');
+    // eslint-disable-next-line no-console
+    console.log('Configuration file written in current directory.'); // NOSONAR
 
     // Copy config.json to the public directory
     try {
       fs.copyFileSync(currentConfigFilePath, outConfigFilePath);
-      console.log('Configuration file copied to public directory.');
+      // eslint-disable-next-line no-console
+      console.log('Configuration file copied to public directory.'); // NOSONAR
```

Comment:
same is the necessary

Reply:
This is only during the development, we can keep this.